### PR TITLE
Chapter 9 Exercise 10 Reworded ambiguity

### LINKED ISSUE #99: Chapter 9 Excercise 10
Error reported in course thinkcspy on page https://runestone.academy/runestone/static/thinkcspy/Strings/Exercises.html by user GrandT007 grandt007@gmail.com
In Chapter 9 excercise 10, the tests provided show that the problem has been solved for overlapping substrings, instead of non-overlapping substrings as stated in the problem

### PR DESCRIPTION
**Pull request by Bryar-Frank**
fixes #99 

It seems that 2 years ago, the problem in the Unit Test was fixed, but the question was still ambiguous. Also there was only one line testing if the function counted only non-overlapping, so we added another line.

### CODE REVIEW

File: _sources/Strings/Exercises.rst [hunk 1]
```diff
@@ -75,17 +75,28 @@ Exercises
...
+                def count(p):
+                    # your code here
+
+                ====
+                from unittest.gui import TestCaseGui
+
+                class myTests(TestCaseGui):
+                    def testOne(self):
+                        string1 = "e"
+                        self.assertEqual(count(string1), 1, "Just one e")
```
Comment (bnmnetp) [inline _sources/Strings/Exercises.rst:95]: I would like to see more tests added here. A mixed string for example an empty string, a long string with no e's are some ideas.

File: _sources/Strings/Exercises.rst [hunk 2]
```diff
@@ -75,17 +75,28 @@ Exercises
 
            .. actex:: ex_8_3
 
-              Assign to a variable in your program a triple-quoted string that contains
-              your favorite paragraph of text - perhaps a poem, a speech, instructions
-              to bake a cake, some inspirational verses, etc.
+                Assign to a variable in your program a triple-quoted string that contains
```
Comment (bnmnetp) [inline _sources/Strings/Exercises.rst:78]: Thanks for updating the indentation.  It makes life better for everyone when it is consistent!

Comment (Bryar-Frank) [reply _sources/Strings/Exercises.rst:108]: Thank you! No problem :)

File: _sources/Strings/Exercises.rst (hunk 1 above)
Comment (Bryar-Frank) [reply _sources/Strings/Exercises.rst]: This initially had more tests but I think this was the pull request that had some merge conflicts, I will attempt to revert those changes and submit a new pull request.

**Pull request closed without merging**